pkg/telemetry/simple_gauge.go

```diff
@@ -0,0 +1,44 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+package telemetry
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+// SimpleGauge tracks how many times something is happening.
+type SimpleGauge interface {
```

Comment:
Is that just a `telemetry.Gauge` without the `...string` to avoid garbage or something similar?

Reply:
Yes, `telemetry.Gauge` without the tags, to avoid a mutex and a map lookup under the hood.